src/EventPage/EventPage.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+
+  page: {
+    padding: 10,
+  },
+  title: {
+    marginTop: 60,
+    paddingHorizontal: 15,
+    fontSize: 30,
+    fontWeight: 'bold',
+  },
+  boxContainer: {
+    marginTop: 30,
+    width: 330,
+
+  },
+  box: {
+    padding: 10,
+    marginBottom: 15,
+    alignItems: 'center',
+    backgroundColor: '#F2F2F2',
```

Comment:
Let's keep colors out of local style objects and move them into `src/globals/colors.js` and then import them into files. This makes it easier to maintain if we suddenly need to change colors for some reason